src/components/button/Button.js

```diff
@@ -1 +1,127 @@
 import styled from 'styled-components';
+
+
+// Button
+const Button = (props) => {
+
+      if (props.name === "Outline") {
```

Comment:
can we organize our code similar to our previous repos? https://github.com/nwplus/portal/blob/dev/src/components/Banner.js 

we could define the styled components outside of the functional component to keep it a bit cleaner